### DISCUSSION TITLE
Как настроить блеклист

### DISCUSSION THREAD

**Discussion by ghost** [bol-van/zapret#803] [Q&A] [answered]
Обход ютуба и других сайтов работает только через badseq
Но из за него перестает работать steam, github и reddit
Можно ли как-то настроить блэклист если MODE_FILTER=none ?
Я не пользуюсь хостлистом тк после блокировки cloudflare перестали работать куча сайтов которых нету ни в одном хостлисте

**Comment by ghost** [comment]
Как я понял --hostlist-exclude= для MODE_FILTER=none не работает?

**Reply by bol-van** [reply, marked as answer]
Можно.
MODE_FILTER=hostlist
ipset/zapret-hosts-user.txt сделать пустым. это означает что он как бы отсутствует и все разрешено
и заполнять ipset/zapret-hosts-user-exlclude.txt
